python/tests/unit/arcticdb/version_store/test_resample.py

```diff
@@ -503,4 +505,94 @@ def test_resampling_empty_type_column(lmdb_version_store_empty_types_v1):
     q = QueryBuilder()
     q = q.resample("s").agg({"col": "first"})
     with pytest.raises(SchemaException):
-        lib.read(sym, query_builder=q)
\ No newline at end of file
+        lib.read(sym, query_builder=q)
+
+class TestResamplingOffset:
```

Comment:
We should also test using `pd.Timedelta` for the offset argument

Reply:
Cool, I didn't know you can parametrize the whole suite.